build.gradle

```diff
@@ -3,3 +3,8 @@ plugins {
     id "com.github.johnrengelman.shadow" version "5.1.0"
 }
 
+allprojects {
```

Comment:
Should this be 1.11 or 11 now? As the java base image is pointing to jdk-11 ?

Reply:
Have to keep compatibility with 1.8 unfortunately. Jenkins seems to be running it so when I changed it, it wouldn't compile in the build pipeline